src/utils/components/ErrorWindow.vue

```diff
@@ -40,7 +44,9 @@ const emit = defineEmits(['close'])
             class="card-header d-flex align-items-center justify-content-sm-end"
             data-cy="window-header"
         >
-            <span v-if="title" class="me-auto text-truncate">{{ i18n.t(title) }}</span>
+            <span v-if="title" class="me-auto text-truncate"
+                >{{ i18n.t(title) }}{{ errorCount }}</span
```

Comment:
would prefer to have something like

```
<script>
// ...
const errorCount = computed(() => store.state.ui.errors.size)
// ...
</script>

<template>
// ...
 <span v-if="title" class="me-auto text-truncate">
    {{ i18n.t(title) }}
    <span v-if="errorCount > 1">({{ errorCount }})</span>
</span>
// ...
</template>
```

Keeping templating stuff (parenthesis around, choice to show it or not) located in the template, and not use some string construction + ternary operator to achieve this

Reply:
yeah, it would be cleaner :)